sonar-db/src/test/java/org/sonar/db/charset/PostgresCharsetHandlerTest.java

```diff
@@ -48,69 +45,76 @@
   @Rule
   public ExpectedException expectedException = ExpectedException.none();
 
-  SqlExecutor selectExecutor = mock(SqlExecutor.class);
-  PostgresCharsetHandler underTest = new PostgresCharsetHandler(selectExecutor);
+  private SqlExecutor selectExecutor = mock(SqlExecutor.class);
+  private PostgresCharsetHandler underTest = new PostgresCharsetHandler(selectExecutor);
 
   @Test
-  public void checks_that_column_is_utf8() throws Exception {
-    answerSql(asList(
+  @Ignore
+  public void upgrade_checks_that_default_collation_and_columns_are_utf8() throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Fix or remove this skipped unit test [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1607)